### DISCUSSION TITLE
tailwindcss doesn't work

### DISCUSSION THREAD

**Discussion by HuangKaiSong** [tailwindlabs/tailwindcss#11823] [Help]
[click here](https://codesandbox.io/p/sandbox/kind-elgamal-r537gx)
I configured tailwindcss according to the documentation, but it didn't work

**Comment by wongjn** [comment]
It seems to be not working because you are using [ESM module](https://nodejs.org/api/esm.html) syntax in `tailwind.config.js` with Tailwind `v3.2`. ESM support was added in [`v3.3.0`](https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.3.0).

You could consider using [CommonJS syntax](https://nodejs.org/api/modules.html) in `tailwind.config.js` like [in this CodeSandbox](https://codesandbox.io/p/sandbox/https-github-com-tailwindlabs-tailwindcss-discussions-11823-9528hx) or upgrade to `v3.3.0` or newer.
